src/component/input.js

```diff
@@ -20,19 +20,32 @@ const baseStyles = StyleSheet.create({
   },
 });
 
-export const TextInput = ({ style, ...props }) => (
-  <RNTextInput
-    style={[baseStyles.input, style]}
-    autoCorrect={false}
-    autoCapitalize="none"
-    underlineColorAndroid="rgba(0,0,0,0)"
-    placeholderTextColor={color.greyPlaceholder}
-    {...props}
-  />
-);
+export class TextInput extends Component {
+  componentDidUpdate() {
+    const { autoFocus, value } = this.props;
+    value.length === 0 && autoFocus && this._input.focus();
+  }
```

Comment:
Cleaned this up a bit in https://github.com/lightninglabs/lightning-app/pull/781/commits/f8010e922c04e7e7adbe40be6b72a82286e7ca02